catroid/src/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -1120,4 +1120,9 @@ public static boolean deleteDirectory(File path) {
 		}
 		return path.delete();
 	}
+
```

Comment:
That won't work, because the wrong BroadcastMessage won't cause the super.fromXML(file)-method (line 171) to throw a ConversionException. Not the tags are wrong, just the value. It should work right under line 170, but then it's called every time the program will be opened after switching programs.